0031-dynamic-configuration-protocol/0031-dynamic-configuration-protocol.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Interledger Dynamic Configuration Protocol
+draft: 1
+---
+
+# Interledger Dynamic Configuration Protocol (ILDCP) v1
+
+## Prerequisites
+This specification assumes the reader is familiar with the following documents:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+- [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)
+
+## Terminology
+
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver. 
+- Nodes are connected to each other and the relationship between two nodes is either:
+  - `parent` and `child` (one node is the `parent` of another node that is relatively its `child`), or,
+  - `peer` and `peer` (two nodes are peered with one another).
+
+## Overview
+Interledger is a protocol suite that consists of several protocols including the [Bilateral Transfer Protocol](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md), the Route Broadcasting Protocol and others necessary for establishment, operation and maintenance of the network. Interledger Dynamic Configuration Protocol (ILDCP) is one of these protocols.
+
+In order to participate in the network a node must have an ILP address. This address is part of a heirarchical address space where child nodes MAY be allocated addresses within the address space of their parent node. This makes routing on the network more efficient than if all nodes had top-level addresses.
+
+A node is therefor either configured with an address which it broadcasts to its peers or, if the node is a child of a parent peer-node, the ILP address of the child node can be allocated to it by the parent. This is done using ILDCP.
```

Comment:
if the node is a child of a parent peer-node -> if the node has a parent in the network hierarchy